datatypes/src/operations/reproject.rs

```diff
@@ -247,7 +247,7 @@ where
 {
     type Out = A;
     fn reproject_clipped(&self, projector: &P) -> Result<A> {
-        const POINTS_PER_LINE: i32 = 7;
+        const POINTS_PER_LINE: i32 = 21;
```

Comment:
how did you decide on this number? Or was it just for debugging purposes? It doesn't influence the reproject_clipped or does it?

Reply:
i think 7 is to small but i changed it back